src/Loading/sceneLoader.ts

```diff
@@ -976,4 +978,143 @@ export class SceneLoader {
             }, pluginExtension);
         });
     }
+
+    /**
+     * Import animations from a file into a scene
+     * @param rootUrl a string that defines the root url for the scene and resources or the concatenation of rootURL and filename (e.g. http://example.com/test.glb)
+     * @param sceneFilename a string that defines the name of the scene file or starts with "data:" following by the stringified version of the scene or a File object (default: empty string)
+     * @param scene is the instance of BABYLON.Scene to append to (default: last created scene)
+     * @param targetConverter defines a function used to convert animation targets from loaded scene to current scene (default: search node by name)
+     * @param onSuccess a callback with the scene when import succeeds
+     * @param onProgress a callback with a progress event for each file being loaded
+     * @param onError a callback with the scene, a message, and possibly an exception when import fails
+     */
+    public static ImportAnimations(rootUrl: string, sceneFilename: string | File = "", scene: Nullable<Scene> = EngineStore.LastCreatedScene, targetConverter: Nullable<(target: any) => any> = null, onSuccess: Nullable<(scene: Scene) => void> = null, onProgress: Nullable<(event: SceneLoaderProgressEvent) => void> = null, onError: Nullable<(scene: Scene, message: string, exception?: any) => void> = null): void {
+        if (!scene) {
+            Logger.Error("No scene available to load animations to");
+            return;
+        }
+
+        let _targetConverter = targetConverter ? targetConverter : this._defaultTargetConverter(scene);
+
+        let onAssetContainerLoaded = (container: AssetContainer) => {
+            SceneLoader.MergeAnimations(scene, container, _targetConverter);
+
+            container.dispose();
+
+            if (onSuccess) {
+                onSuccess(scene);
+            }
+        };
+
+        // Reset, stop and dispose all animations before loading new ones

```

Comment:
This should be optional

Reply:
Ok

Reply:
We have added the following parameters to the API:
- '**overwriteAnimations**' when true, animations are cleaned before importing new ones. Animations are appended otherwise
- '**animationGroupLoadingMode**' defines how to handle old animations groups before importing new ones

The resulting UI is:

![2019-12-04_19h32_01](https://user-images.githubusercontent.com/20560340/70170818-c1938a00-16cd-11ea-8f21-3e8ac07034a3.png)

and this when appending animations:
![2019-12-04_19h32_52](https://user-images.githubusercontent.com/20560340/70170832-c8220180-16cd-11ea-9be7-29560d4008b7.png)

I recognize that the label could be better :-)

The options for **animations groups** are:
- CLEAN : Reset all old animations to initial state then dispose them.
- STOP : Stop all old animations.
- SYNC : Restart old animations from first frame.
- NOSYNC : Old animations remains untouched.